core/server/data/validation/index.js

```diff
@@ -131,24 +128,17 @@ validateSettings = function validateSettings(defaultSettings, model) {
 };
 
 validateActiveTheme = function validateActiveTheme(themeName) {
```

Comment:
Could we move the theme validation to the new theme package?
To have one single place for all kinds of theme handling.

e.g. `core/server/themes/validation`